packages/react-query/src/index.ts

```diff
@@ -1,5 +1,36 @@
+import { AnyProcedure, inferProcedureInput } from '@trpc/server';
+import { QueryType, getArrayQueryKey } from './internals/getArrayQueryKey';
+import { getQueryKey as $getQueryKey } from './internals/getQueryKey';
+import { DecorateProcedure } from './shared';
+
 export * from '@trpc/client';
 
 export { createTRPCReact, type CreateTRPCReact } from './createTRPCReact';
 export { createReactQueryHooks } from './interop';
 export type { inferReactQueryProcedureOptions } from './utils/inferReactQueryProcedure';
+
+/**
+ * Method to extract the query key for a procedure
+ * @param type - defaults to `any`
+ * @link https://trpc.io/docs/useContext#-the-function-i-want-isnt-here
+ */
+export function getQueryKey<
```

Comment:
place somewhere else and just use this as an export-entrypoint?

Reply:
oh god woops

Reply:
admit it you just copy-pasted my commit from here https://github.com/trpc/trpc/commit/fb9858b8166471be1379382cf2e4928fb43e3f54 🤣

Reply:
oh.. i def did but i changed the types :p

```ts
  procedure: DecorateProcedure<TProcedure, TFlags, TPath>,
  ..._params: inferProcedureInput<TProcedure> extends undefined
    ? []
    : [input: inferProcedureInput<TProcedure>, type?: QueryType]
) {
```

Reply:
🐐